apps/web-giddh/src/app/shared/header/header.component.ts

```
@@ -260,6 +260,8 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
     public isProdMode: boolean = PRODUCTION_ENV;
     /** True if consolidated branch */
     public isConsolidatedBranch: boolean;
+    /** Holds true if plan is either trail or cancelled */
+    public isCurrentSubscriptionTrialOrCancelled: Boolean = null;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)